apps/ledger-live-mobile/src/screens/SendFunds/02-SelectRecipient.tsx

```diff
@@ -38,6 +39,7 @@ import { ScreenName } from "../../const";
 import { accountScreenSelector } from "../../reducers/accounts";
 import DomainServiceRecipientRow from "./DomainServiceRecipientRow";
 import RecipientRow from "./RecipientRow";
+import { EditOperationCard } from "../../families/ethereum/EditTransaction/EditOperationCard";
```

Comment:
To include ethereum specific components in common part is an anti pattern for our coin modulation. I think you should move the flow into eth family.
We can discuss it offline if you want.